monitor/documentation/src/main/java/org/datacleaner/documentation/swagger/SecuritySchemeObject.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.documentation.swagger;
+
+import java.util.HashMap;
+
+/**
+ * Added by jakub on 1.3.16
+ */
+public class SecuritySchemeObject extends HashMap {
```

Comment:
And should we not add stuff to avoid compiler warnings ... such as generic arguments and the serialVersionUID?